packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -632,25 +635,51 @@ Posts.addView("scheduled", (terms: PostsViewTerms) => ({
  * @summary Draft view
  */
 Posts.addView("drafts", (terms: PostsViewTerms) => {
-  let query = {
+  let query: any = {
```

Comment:
This is `ViewQueryAndOptions<Posts>`

Reply:
(The way to find out is using IDE hover-over on the `Posts.addView` call, which should show you what type of function it's expecting for the second argument)

Reply:
It would accept ViewQueryAndOptions<any>. ViewQueryAndOptions<Posts> is not valid and using any different fragment name got lint errors with the `viewFieldAllowAny` below.

Reply:
Ah, nope. I get lint errors even with that.